redfish-core/lib/pcie_slots.hpp

```diff
@@ -90,6 +90,55 @@ inline std::string analysisSlotType(const std::string& slotType)
     return "";
 }
 
+inline void
+    getLinkedPcieDevice(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                        const std::string& slotPath, size_t index)
+{
+    // Collect device associated with this slot and
+    // populate it here
+    crow::connections::systemBus->async_method_call(
+        [asyncResp, index](
+            const boost::system::error_code ec,
+            const std::vector<std::pair<
+                std::string,
+                std::vector<std::pair<std::string, std::vector<std::string>>>>>&
+                subtree) {
+            if (ec)
+            {
+                BMCWEB_LOG_ERROR << "D-Bus response error on GetSubTree " << ec;
+                messages::internalError(asyncResp->res);
+                return;
+            }
+            if (subtree.size() == 0)
+            {
+                BMCWEB_LOG_ERROR
```

Comment:
this isn't an error is it?  you can have an empty slot?

Reply:
Fixed. Changed to Debug

Reply:
Even demoted to debug I don't really think tracing is appropriate here.  There are plenty of other ways to figure out if a slot is empty or not.